gapic/schema/wrappers.py

```diff
@@ -767,6 +767,29 @@ def http_opt(self) -> Optional[Dict[str, str]]:
         # TODO(yon-mg): enums for http verbs?
         return answer
 
+    @property
+    def path_params(self) -> Sequence[str]:
+        """Return the path parameters found in the http annotation url"""
+        # TODO(yon-mg): fully implement grpc transcoding (currently only handles basic case)
+        if self.http_opt is None:
+            return []
+        pattern = r'\{\w+\}'
+
+        return [x[1:-1] for x in re.findall(pattern, self.http_opt['url'])]
```

Comment:
We can move more of the logic into the regex by using capturing.
```python
param_re = r"\{(\w+)\}"
return re.findall(param_re, self.http_opt['url']) if self.http_opt else []
```

Reply:
good idea